LoopKit/TemporaryScheduleOverride.swift

```diff
@@ -50,14 +50,27 @@ public struct TemporaryScheduleOverride: Hashable {
     public var startDate: Date
     public let enactTrigger: EnactTrigger
     public let syncIdentifier: UUID
+    
+    public var actualEnd: End = .natural
```

Comment:
I moved this logic out of TemporayScheduleOverrideHistory and into here so the history screens can have access to it

Reply:
Can OverrideEvent be cleaned up to not need this then, since OverrideEvent contains a TemporaryScheduleOverride?

Reply:
Yes & done!